Configuration/Template.php

```diff
@@ -158,4 +165,22 @@ public function allowArray()
     {
         return false;
     }
+
+    /**
+     * @param array $owner
+     */
+    public function setOwner(array $owner)
```

Comment:
As far as I can tell, this is never triggered if it's not an array: https://github.com/sensiolabs/SensioFrameworkExtraBundle/blob/master/EventListener/ControllerListener.php#L51-L55